src/Frontend/Modules/Blog/Layout/Templates/Archive.html.twig

```diff
@@ -41,7 +41,7 @@
             {% for item in items %}
               <tr>
                 <td class="date">
-                  <time itemprop="datePublished" datetime="{{ item.publish_on|date('Y-m-d\TH,i,s' ) }}">
+                  <time itemprop="datePublished" datetime="{{ item.publish_on|date('Y-m-d\\TH:i:s.vP' ) }}">
```

Comment:
do there need to be 2 backslashes?

Reply:
Yes, for some reason, if you use `\T` if will output something like `CEST`.